src/Workleap.DomainEventPropagation.Publishing/Workleap.DomainEventPropagation.Publishing.csproj

```diff
@@ -11,7 +11,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Messaging.EventGrid" Version="4.21.0" />
+    <PackageReference Include="Azure.Messaging.EventGrid" Version="4.22.0-beta.1" />
```

Comment:
Coquin

Reply:
Turns out that we might be capable of using a non-beta version because this package received an update about 2 weeks ago with `4.22.0` 👀

Reply:
Unfortunately, `4.22.0` doesn't include what we need for CloudEvents publishing, so this will have to wait 😢